editor/src/clj/editor/cljfx_form_view.clj

```diff
@@ -393,7 +393,7 @@
                     {:text (value->label x)})
      :cell-factory (fn [x]
                      {:text (value->label x)})
-     :items (sort (mapv first options))}))
+     :items (mapv first options)}))
```

Comment:
Values in dropdowns were always sorted when editing a material. I kind of want the None values to come first. Also, if you want your items sorted, you should sort them yourself and not rely on the view to do it.